backend/src/main/java/com/bakdata/conquery/models/query/concept/specific/CQConcept.java

```diff
@@ -64,8 +66,9 @@
 @ToString
 public class CQConcept extends CQElement implements NamespacedIdHolding {
 
-	@Valid @NotEmpty
-	private List<ConceptElementId<?>> ids = Collections.emptyList();
+	@Valid @NotEmpty @NsIdRefCollection
+	private List<ConceptElement<?>> ids = Collections.emptyList();
```

Comment:
Sind keine Ids mehr

Reply:
habe es umbenannt in `elements` und ein JsonProperty drauf gesetzt für ids